### PR TITLE
bump v1.6 rc.0, fix integration tests

### PR DESCRIPTION
**Pull request by ca-scribner**
This PR bumps the workload to v1.6, as well as (attempts to) fix integration tests

### CODE REVIEW

**Comment by ca-scribner** [comment]
I'm stuck on the integration test fixes.  v1.6's dashboard removed the ID (#newResource) that was on the button we want to press, so now I'm at a loss.  I think the tests are working or nearly working apart from that issue.  I also think the charm is working with volumes v1.6, although its only been lightly manually tested

File: tox.ini [hunk 1]
```diff
@@ -20,12 +20,12 @@ deps =
 
 [testenv:integration]
 deps =
-    git+https://github.com/juju/python-libjuju.git
-    lightkube
+    juju
```
Comment (DnPlas) [inline tox.ini:23]: What is the reason for this?

File: tox.ini [hunk 2]
```diff
@@ -20,12 +20,12 @@ deps =
...
     pytest
     pytest-operator
     selenium
     selenium-wire
+    lightkube<0.11
```
Comment (DnPlas) [inline tox.ini:28]: Should we have a test-requirements.txt?

**Review by DnPlas — COMMENTED** [review]
Thanks @ca-scribner , couple of questions, other than that LGTM.

File: config.yaml [hunk 3]
```diff
@@ -12,5 +12,5 @@ options:
     description: Which mode out of `development` or `production` the backend server should use
   secure-cookies:
     type: boolean
-    default: false
+    default: true
```
Comment (ca-scribner) [inline config.yaml:15]: I should leave this as `false`.  We in the past had troubles with this setting
```suggestion
    default: false
```

File: tox.ini (hunk 1 above)
Comment (ca-scribner) [reply tox.ini:23]: Good question, but really I shouldn't do it here anyway.  I'll do this in a separate PR

File: tox.ini (hunk 2 above)
Comment (ca-scribner) [reply tox.ini:28]: good question.  We seem inconsistent on this - I think we sometimes have either.  Maybe we should bring this up as a general code style thing that we make a team rule for?

**Comment by ca-scribner** [comment]
This is good to merge, unless we want to address [@dnplas' comment](https://github.com/canonical/kubeflow-volumes-operator/pull/31#discussion_r921714138)

**Review by DomFleischmann — APPROVED** [review]

**Pull request merged**